etcdserver/server.go

```diff
@@ -1292,8 +1294,6 @@ func (s *EtcdServer) apply(es []raftpb.Entry, confState *raftpb.ConfState) (appl
 func (s *EtcdServer) applyEntryNormal(e *raftpb.Entry) {
 	shouldApplyV3 := false
 	if e.Index > s.consistIndex.ConsistentIndex() {
-		// set the consistent index of current executing entry
-		s.consistIndex.setConsistentIndex(e.Index)
```

Comment:
How is consistent index used in the store and v3 applier? Initially, I thought consistent index is used to keep track of latest applied entry, and it is used to guard v3 applier so that we don't re-apply applied entries.